src/OAuth/OAuth2/Service/Facebook.php

```diff
@@ -151,13 +151,23 @@ public function getAccessTokenEndpoint()
      */
     protected function parseAccessTokenResponse($responseBody)
     {
+        // omg... if error, $responseBody contains JSON
+        // e.g: {"error":{"message":"Missing redirect_uri parameter.","type":"OAuthException","code":191}}
+        // but not sure that it is always
+        $data = @json_decode($responseBody, true);
```

Comment:
Does `json_decode` trigger an error or just return `null`? I'd like to avoid error suppression if it's not absolutely needed.